infra/core/host/appservice.bicep

```diff
@@ -0,0 +1,123 @@
+metadata description = 'Creates an Azure App Service in an existing Azure App Service plan.'
```

Comment:
Please do a diff with azure-dev/infra/core and this and see if there are any changes.